Source/WebCore/platform/graphics/transforms/PerspectiveTransformOperation.h

```diff
@@ -77,9 +75,9 @@ class PerspectiveTransformOperation final : public TransformOperation {
 
     void dump(WTF::TextStream&) const final;
 
-    PerspectiveTransformOperation(const std::optional<Length>&);
+    PerspectiveTransformOperation(const std::optional<float>&);
 
-    std::optional<Length> m_p;
+    std::optional<float> m_p;
```

Comment:
If memory use is a possible issue, should note that `Markable` works quite well for `float`. I think we have a “store as `Markable` use `std::optional` for arguments and function results” pattern? Not saying we should use it here, though.

Reply:
It won’t make any difference here due to minimum allocation size I believe.